File: 3 КУРС/2 семестр/Java/JavaProects/app/src/main/java/com/example/voiceassistent/NumberService.java
package com.example.voiceassistent;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class NumberService {
    public static NumberAPI getApi() {
        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl("https://htmlweb.ru") //Базовая часть адреса
                .addConverterFactory(GsonConverterFactory.create().create()) //Конвертер, необходимый для преобразования JSON'а в объекты
                .build();

        return retrofit.create(NumberAPI.class); //Создание объекта, при помощи которого будут выполняться запросы
    }

}

File: 3 КУРС/2 семестр/Учебная практика/JavaProects/app/src/main/java/com/example/voiceassistent/NumberService.kt
package com.example.voiceassistent

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object NumberService {
    val api: NumberAPI
        get() {
            val retrofit = Retrofit.Builder()
                    .baseUrl("https://htmlweb.ru") //Базовая часть адреса
                    .addConverterFactory(GsonConverterFactory.create()) //Конвертер, необходимый для преобразования JSON'а в объекты
                    .build()
            return retrofit.create(NumberAPI::class.java) //Создание объекта, при помощи которого будут выполняться запросы
        }
}